docs/workflow/building/coreclr/cross-building.md

```diff
@@ -11,9 +11,13 @@ Install the ARM tools and Windows SDK, as described [here](https://github.com/do
 Cross compiling CoreCLR
 -----------------------
 
-Build using "-arm" as the architecture. For example:
+Build using "arm" as the architecture. For example:
 
-    C:\runtime> build.cmd -arm -debug
+    C:\runtime> .\build.cmd -subsetCategory CoreClr -subset Runtime -arch arm -c debug
```

Comment:
```suggestion
    C:\runtime> build.cmd -subsetCategory CoreClr -subset Runtime -arch arm -c debug
```

It is unusual to have `.\` on Windows.

Reply:
Well, this is required by PowerShell.

Reply:
We should have the docs consistent. There is one outlier in the existing docs that has build.cmd with `.\`, the rest has it without it.

Prepending .\ is a second nature for Powershell users, not sure whether we need to have it prepended in the docs.